src/EmployeeManagement/EmployeeManagement.IntegrationTest/ControllerTests/EmployeeControllerTests.cs

```diff
@@ -23,16 +22,16 @@ public async Task ShouldGetEmployeeList_WhenCalled_GetAllEmployeesAsync()
         {
             //Arrange
             TestDBSampleData.InitializeDbForTests(_appFactory);
-            var employeeDto = _mapper.Map<List<EmployeesDTO>>(TestDBSampleData.FetchSeedingEmployees());
+            var employeeDto = _mapper.Map<List<EmployeeDTO>>(TestDBSampleData.FetchSeedingEmployees());
```

Comment:
I think it will be better to use strongly typed both from and to types in mapper.

Reply:
Updated to strongly typed